### PR TITLE
Add typescript templates

### PR DESCRIPTION
**Pull request by zackdotcomputer**
Adds two templates showing how to use this component in typescript with either the vanilla tsc or with webpack as your compiler/bundler.

This was done to explore whether it was even possible after reading #12

### CODE REVIEW

File: templates/express-starter-typescript-tsc/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+app: express-starter
+component: express
+name: express-starter
```
Comment (eahefnawy) [inline templates/express-starter-typescript-tsc/serverless.yml]: could you change that to `express-starter-typescript-tsc`?

File: templates/express-starter-typescript-webpack/serverless.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+app: express-starter
+component: express
+name: express-starter
```
Comment (eahefnawy) [inline templates/express-starter-typescript-webpack/serverless.yml]: could you change that to `express-starter-typescript-webpack`?

**Review by eahefnawy — CHANGES_REQUESTED** [review]
Thanks for the PR! DId you face any issues getting it to work with TypeScript?

**Comment by zackdotcomputer** [comment]
@eahefnawy Not at all - was able to pipe the existing templates through tsc or webpack just fine and, once I added the typing, everything seemed to work ok.

Made the changes you requested and fixed the lint/prettier issues that the GH Action turned up

**Review by eahefnawy — APPROVED** [review]
Thank you 🙏

**Pull request merged**